contracts/near/eth-client/src/tests.rs

```diff
@@ -262,8 +262,46 @@ fn add_dags_merkle_roots() {
     assert_eq!(dmr.dag_merkle_roots[0], contract.dag_merkle_root(0));
     assert_eq!(dmr.dag_merkle_roots[10], contract.dag_merkle_root(10));
     assert_eq!(dmr.dag_merkle_roots[511], contract.dag_merkle_root(511));
+    assert_eq!(dmr.dag_merkle_roots[699], contract.dag_merkle_root(699));
 
-    let result = catch_unwind_silent(|| contract.dag_merkle_root(512));
+    let result = catch_unwind_silent(|| contract.dag_merkle_root(700));
+    assert!(result.is_err());
+}
+
+#[test]
+#[cfg_attr(feature = "eip1559", ignore)]
```

Comment:
Any reason to ignore it? Is it relevant for the test?

```suggestion
#[cfg_attr(feature = "eip1559", ignore)]
```

Reply:
Yes it is relevant due to the init call

Reply:
Well, it shouldn't be if you change the block data provided there

Reply:
93c42df804f26262f3dd7a798f51a54db54eafd2